scripts/cmux-workspace-handoff-lab.sh

```
@@ -0,0 +1,346 @@
+          printf '\nCodex Arm A (real HOME, different cwd): exit=%s; rollout=%s; bytes=%s→%s; which rollout grew=%s\n' "$CODEX_A_STATUS" "$CODEX_FILE" "$CODEX_A_BEFORE_SIZE" "$CODEX_A_AFTER_SIZE" "$([[ "$CODEX_A_AFTER_SIZE" -gt "$CODEX_A_BEFORE_SIZE" ]] && echo "$CODEX_FILE" || echo 'none observed')" >>"$FINDINGS"
+          if ((CODEX_A_STATUS == 0)) && grep -qi "amber-$RUN_ID" "$LOGS/codex-arm-a.summary" && grep -q "cwd=/" "$LOGS/codex-arm-a.summary"; then
+            pass 'codex Arm A recalled codeword from different cwd'
+            printf '| codex Arm A | PASS | real HOME; codeword/cwd summary and rollout growth recorded above |\n' >>"$FINDINGS"
+          else
+            fail 'codex Arm A resume'
+            printf '| codex Arm A | REAL-BLOCKED/FAIL | exit %s; exact output/error retained in logs/codex-arm-a.* |\n' "$CODEX_A_STATUS" >>"$FINDINGS"
+          fi
+          # Keep Arm A's destination for inspection, then reuse the required
+          # codex-dst path for Arm B's real-home restore.
+          mv "$RUN_DIR/codex-dst" "$RUN_DIR/codex-dst-arm-a"
+          CODEX_REL="${CODEX_FILE#$HOME/.codex/}"
```

Shellcheck (0.11.0)

[info] Expansions inside ${..} need to be quoted separately, otherwise they match as patterns.

(SC2295)